.pre-commit-config.yaml

```diff
@@ -1,3 +1,4 @@
+# Copyright © 2024 ANSYS, Inc. Unauthorized use, distribution, or duplication is prohibited. 
```

Comment:
To piggyback on Koninckx's comment, I don't think this should be public. Is the header necessary here?

Also applies to `.protolint.yaml` and `pyproject.toml`.

Reply:
We are supposed to add this header to all of our code, public or private.

Reply:
Yes, we have to add it even for open source project.